src/components/DpInput/DpInput.stories.mdx

```diff
@@ -76,7 +76,7 @@ If an input field is `required`, an asterisk will be added to its label.
             label: {
                 text: 'Phone'
             },
-            pattern: '^(\+?)(-| |[0-9]|\(|\))*$',
+            pattern: '^(\\+?)(-| |[0-9]|\\(|\\))*$',
```

Comment:
Why these changes?

Reply:
I hava this error msg `Unable to check <input pattern='^(+?)(-| |[0-9]|(|))*$'> because the pattern is not a valid regexp: nothing to repeat`